doc/nrf/known_issues.rst

```diff
@@ -411,6 +437,14 @@ NCSDK-18376: P-GPS in external flash fails to inject first prediction during loa
   When using external flash with P-GPS, stored predictions cannot be reliably read to satisfy the modem's need for ephemeris data when actively downloading and storing predictions to the same external flash.
   Once the download is complete, the predictions can be reliably read.
 
+nRF7002
+*******
+
+.. rst-class:: v2-3-0
+
+SHEL-1352: Incorrect base address used in the OTP TX trim coefficients
+  Because of the use of an incorrect base address used in the One-Time Password TX trim coefficients, the transmit power will deviate by +/- 2dB from the target value.
```

Comment:
@bama-nordic , please review.

Reply:
I suggest rewording the description as "Incorrect base address used for TX trim coefficients in the One-Time Programmable (OTP) memory, resulting in transmit power deviating by +/- 2dB from the target value"